jsapp/js/attachments/BulkDeleteMediaFiles.tsx

```diff
@@ -61,36 +68,46 @@ export default function BulkDeleteMediaFiles(props: BulkDeleteMediaFilesProps) {
 
       <Modal opened={opened} onClose={handleCloseModal} title={t('Delete media files')} size={'md'}>
         <FocusTrap.InitialFocus />
-        <Stack>
-          <Checkbox
-            label={
-              <Text>
-                {t('You are about to permanently remove the following media files from the selected submissions:')}
-                <br />
-                {getMediaCount(props.selectedSubmissions)}
-              </Text>
-            }
-            onClick={() => setWarningAcknowledged(!warningAcknowledged)}
-            checked={warningAcknowledged}
-          />
-          <Alert iconName='warning' type='error'>
-            {t('Careful - it is not possible to recover deleted media files')}
-          </Alert>
-          <Group justify='flex-end'>
-            <Button variant='light' size='lg' onClick={close} disabled={isDeletePending}>
-              {t('Cancel')}
-            </Button>
 
-            <Button
-              disabled={!warningAcknowledged || isDeletePending}
-              variant='danger'
-              size='lg'
-              onClick={handleConfirmDelete}
-            >
-              {t('Delete')}
-            </Button>
-          </Group>
-        </Stack>
+        {filteredSubmissions.length === 0 && (
+          // This can only happen if the user has permissions to view (but not delete) certain submissions
+          <Stack>
+            <Text>{t('You do not have permissions to delete the selected submissions.')}</Text>
```

Comment:
```suggestion
            <Text>{t('You do not have sufficient permissions to delete attachments from the selected submissions.')}</Text>
```
What do you think about this? We are deleting attachments here, right? :)

Reply:
And would you also make a screenshot of that modal and ask Tessa what does she thing about it? I am wondering for example if we should use `<Alert` here with yellow color for this message. But I would rather merge this PR and have some additional change requested by Tessa in other PR :)

Reply:
Let's merge it and let Tessa give a more comprehensive review of the UI once it's done